keyboards/ferris/keymaps/ghidalgo93/rules.mk

```diff
@@ -0,0 +1,2 @@
+OLED_DRIVER_ENABLE = no
+EXTRAKEY_ENABLE = yes
```

Comment:
Same here — this file should be removed. Media keys are enabled at the keyboard level, and this keyboard does not even have OLED displays.